main.go

```diff
@@ -9,18 +9,18 @@ import (
 	"github.com/golang/glog"
 	"golang.org/x/oauth2"
 
-	"github.com/google/go-github/v28/github"
+	"github.com/google/go-github/github"
 	"gopkg.in/yaml.v2"
 )
 
 func contains(slice []string, item string) bool {
-    set := make(map[string]struct{}, len(slice))
-    for _, s := range slice {
-        set[s] = struct{}{}
-    }
+	set := make(map[string]struct{}, len(slice))
```

Comment:
I love spacces not tabs and my IDE is set to replace them automatically. I hope you do not mind. I can revert it.